MAINTAINERS.md

```diff
@@ -1,2 +1 @@
-* Krasi Georgiev <kgeorgie@redhat.com> @krasi-georgiev for `api/...`
 * Björn Rabenstein <beorn@grafana.com> @beorn7 for everything else
```

Comment:
The "for everything else" in my line has to go now. :o)

Reply:
yep , sorry , updated.